addons/mail/static/src/models/message_tracking_value.js

```diff
@@ -0,0 +1,182 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one } from '@mail/model/model_field';
+
+import { format } from 'web.field_utils';
+
+registerModel({
+    name: 'MessageTrackingValue',
+    identifyingFields: ['id'],
+    modelMethods: {
+        //----------------------------------------------------------------------
+        // Public
+        //----------------------------------------------------------------------
+        /**
+         * @static
+         * @param {Object} data
+         * @return {Object}
+         */
+        convertData(data) {
```

Comment:
Convert data is deprecated, ideally we shouldn't introduce new one...
It would be time to change the server to actually return commands for tracking if necessary, and directly call update without convert.